contracts/vortex/CarbonVortex.sol

```diff
@@ -652,6 +652,15 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
             _resetTrading(token, 0);
         }
 
+        // if available target token trading amount is below the min target token sale amount, reset the target token auction
+        if (
+            Token.unwrap(_finalTargetToken) != address(0) &&
+            _amountAvailableForTrading(_targetToken) <
+            _minTokenSaleAmounts[_targetToken] / _minTokenSaleAmountMultiplier
```

Comment:
If accuracy is critical here for `_resetTradingTarget` to take place, then you should change the division on the RHS)to a multiplication on the LHS (and of course, cast the value returned from the call `_amountAvailableForTrading` to `uint256`, in order to avoid reverting on overflow).

Reply:
Was thinking about this - but considering the expected values for minTokenSaleAmounts and minTokenSaleMultiplier, I think that RHS division is sufficient